bundle/deploy/metadata.go

```diff
@@ -0,0 +1,18 @@
+package deploy
+
+import "github.com/databricks/cli/bundle/config"
+
+var LatestMetadataVersion = 1
+
+var MetadataTag = "metadata"
```

Comment:
Where is this used?

Reply:
removed, leftover from attempt at using struct tags.